### DISCUSSION TITLE
HUSD Funds not arrived at Solana - pending continues

### DISCUSSION THREAD

**Discussion by liber9** [wormhole-foundation/wormhole#221] [Problems / bugs] [answered]
Hello, 

I tried to transfer HUSD using the Solana Wormhole and the transaction got stuck on "Awaiting completion on Solana" the funds were removed from my ETH wallet and were not deposited in my Solana Wallet.

Here is the ETH transaction:
https://etherscan.io/tx/0x536d30913d7f7eec3f85009998c0a77d8cf4933027aecef0c1508555ff3f13b7

Here is the Solana transaction:
https://explorer.solana.com/tx/3UYagmA8EGZgUeakhMtZ9Jx7Pph2yR7qMTUQ7VAnhoebUDbwnk2dXAcbrbhqcHSrn8kDQA5UqEqEfBr9ExS1ztA5

Solana Wallet Address:
3Kf2yscxHAb9BYV1JRyupRGR2oNJB5aPmWiQm3wyuxSE

Please resolve this problem. 

Let me know if you need any additional information.

Thank you.

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed in https://explorer.solana.com/tx/5HJtaunwgWBs7X6NdX8Sw1MBt2gN2CoHnJX9xUwo2gHyiHmugw3o2muoag44cEUyAeQAcQ1GqRVKGSSYyB5KSMPz
